opencti-platform/opencti-front/src/private/components/data/stream/PublicStreamLines.tsx

```diff
@@ -0,0 +1,217 @@
+import { graphql, loadQuery, useFragment, usePreloadedQuery } from 'react-relay';
+import ListItemText from '@mui/material/ListItemText';
+import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import { toPairs } from 'ramda';
+import Chip from '@mui/material/Chip';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import { CastConnectedOutlined, StopCircleOutlined } from '@mui/icons-material';
+import ListItem from '@mui/material/ListItem';
+import { ListItemSecondaryAction } from '@mui/material';
+import { OpenInNew } from 'mdi-material-ui';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import { environment } from '../../../../relay/environment';
+import { PublicStreamLinesQuery } from './__generated__/PublicStreamLinesQuery.graphql';
+import ListLines from '../../../../components/list_lines/ListLines';
+import ListLinesContent from '../../../../components/list_lines/ListLinesContent';
+import { StreamLineDummy } from './StreamLine';
+import { DataColumns } from '../../../../components/list_lines';
+import { truncate } from '../../../../utils/String';
+import { useFormatter } from '../../../../components/i18n';
+import { Theme } from '../../../../components/Theme';
+import { PublicStreamLines_node$key } from './__generated__/PublicStreamLines_node.graphql';
+import { isNotEmptyField } from '../../../../utils/utils';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 5,
+  },
+  filter: {
+    fontSize: 12,
+    lineHeight: '12px',
+    height: 20,
+    marginRight: 7,
+    borderRadius: 10,
+  },
+  operator: {
+    fontFamily: 'Consolas, monaco, monospace',
+    backgroundColor: theme.palette.background.accent,
+    height: 20,
+    marginRight: 10,
+  },
+  item: {
+    paddingLeft: 10,
+    height: 50,
+  },
+}));
+
+const publicStreamLinesFragment = graphql`
+  fragment PublicStreamLines_node on StreamCollection {
+    id
+    name
+    live
+    description
+    public
+    filters
+  }
+`;
+
+const publicStreamLinesQuery = graphql`
+  query PublicStreamLinesQuery {
+    streamCollections {
```

Comment:
The private stream appear on this component

![image](https://user-images.githubusercontent.com/53513584/214237901-bef87554-7cf5-4fd3-b314-3f8478ac9f70.png)
![image](https://user-images.githubusercontent.com/53513584/214237933-8f5d576c-a936-4eb1-9cd2-1fae078c7c46.png)

Reply:
Done